src/test/scala/strawman/collection/test/BuildFromTest.scala

```diff
@@ -0,0 +1,116 @@
+package strawman
+package collection.test
+
+import org.junit.Test
+import strawman.collection.Iterable
+import strawman.collection.mutable.Builder
+import strawman.collection._
+
+import scala.{Any, Boolean, Either, Int, Left, None, Option, PartialFunction, Right, Some, Unit}
+import scala.Predef.{ArrowAssoc, implicitly}
+import scala.math.Ordering
+import java.lang.String
+
+class BuildFromTest {
```

Comment:
And you should be able to do the same with the current `CanBuildFrom`. It certainly works with the new `BuildFrom`. I have examples of both styles in my PR.